.gitignore

```diff
@@ -47,3 +47,6 @@ profile.cov
 /dashboard/assets/package-lock.json
 
 **/yarn-error.log
+
```

Comment:
adding this will lead to "all future changes `.github` folder to will also be ignored"

Reply:
deleted.